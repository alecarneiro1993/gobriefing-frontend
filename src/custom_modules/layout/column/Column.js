import React from "react";
import PropTypes from "prop-types";
import Grid from "@material-ui/core/Grid";

export default function Column({ children, ...rest }) {
  return (
    <Grid item {...rest}>
      {children}
    </Grid>
  );
}

Column.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired
};
