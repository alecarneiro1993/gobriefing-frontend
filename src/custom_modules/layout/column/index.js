import React from "react";
import PropTypes from "prop-types";
import { Grid } from "@material-ui/core";

export function Column({ children, ...props }) {
  return (
    <Grid item {...props}>
      {children}
    </Grid>
  );
}

Column.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired
};
