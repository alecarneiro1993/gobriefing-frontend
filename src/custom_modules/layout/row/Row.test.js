import React from "react";
import { shallow } from "enzyme";

import { Row } from ".";
import { Column } from "../column";

describe("Row", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(
      <Row>
        <Column>Test</Column>
      </Row>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
