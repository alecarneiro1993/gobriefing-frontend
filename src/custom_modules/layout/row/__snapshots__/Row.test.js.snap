// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders without crashing 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
>
  <Column>
    Test
  </Column>
</WithStyles(ForwardRef(Grid))>
`;
